ui/app/routes/jobs/run/index.js

```diff
@@ -67,6 +70,17 @@ export default class RunRoute extends Route {
     notifyForbidden(this)(e);
   }
 
+  async establishDefaultJobTemplates() {
+    // Iterates over DEFAULT_JOB_TEMPLATES and creates a variable record for each of them, if they dont already exist
+    return await Promise.all(
+      DEFAULT_JOB_TEMPLATES.map((template) => {
+        if (!this.store.peekRecord('variable', template.id)) {
+          return this.store.createRecord('variable', template);
+        }
+      })
+    );
```

Comment:
At this stage, scaffold all our default job templates as Variable objects, so they get access to all the goodness (.items, etc.) that the model provides.